Doc/howto/clinic.rst

```diff
@@ -539,7 +539,17 @@ Let's dive in!
         };
 
 
-16. Compile, then run the relevant portions of the regression-test suite.
+16. Code generated by Argument Clinic might use ``_Py_ID`` macro inside.
+
+    Sample::
+
+        &_Py_ID(new_unique_py_id)
+
+    It means that you would need to run ``Tools/scripts/generate_global_objects.py``
```

Comment:
If it does, you'll have to run ``Tools/scripts/generate_global_objects.py`` to regenerate the list of precompiled identifiers at this point.

Reply:
Thanks, done!

It is always great to have native-speakers to double check things like this. I think in another language and it does not translate that well in some cases :)